distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -221,6 +220,13 @@ org.codice.ddf.http.access.log.enabled=false
 #
 org.codice.ddf.security.ecp.enabled=true
 
+#
+# IDP (Identity Provider) Settings
+#
+# How long IDP clients should cache the IDP's metadata, in milliseconds. Default is 7 days.
+org.codice.ddf.security.saml.Metadata.cacheDuration=86400000
```

Comment:
❓ This is 1 day in milliseconds, should it be 7?

Reply:
Good catch.